ClavaLaraApi/src-lara-clava/clava/clava/code/Inliner.js

```diff
@@ -0,0 +1,107 @@
+laraImport("clava.ClavaJoinPoints");
+
+class Inliner {
+  #variablePrefix;
+  #variableIndex;
+  constructor() {
+    this.#variableIndex = 0;
+    this.#variablePrefix = "__inline";
+  }
+
+  #getInlinedVarName(originalVarName) {
+    return `${this.#variablePrefix}_${this.#variableIndex}_${originalVarName}`;
+  }
+
+  inline($exprStmt) {
```

Comment:
If we want to recursively inline calls inside calls inside calls, the intent of this class is to be used as a building block for that?